### PR TITLE
Deprecate cocoapods and migrate to SPM

### PR DESCRIPTION
**Pull request by danialepaco**
#### Description:

* Remove cocoapods from the project
* Add support for SPM
* Update README
* Update Init script

### CODE REVIEW

File: ios-base/Info.plist [hunk 1]
```diff
@@ -39,6 +39,8 @@
 	<string></string>
 	<key>FacebookKey</key>
 	<string>$(FACEBOOK_KEY)</string>
+	<key>FacebookClientToken</key>
+	<string></string>
```
Comment (kstoletniy) [inline ios-base/Info.plist:43]: i think we have deprecated the facebook feature, so you can remove it

**Review by kstoletniy — APPROVED** [review]
it seems to be everything ok, however did you perform some quick test if everything if working?

**Comment by glm4** [comment]
Great work @danialepaco !
I think you also need to update the init executable.
To create a binary executable from the init.swift script you can run:
`xcrun swiftc init.swift`

File: ios-base/Info.plist (hunk 1 above)
Comment (danialepaco) [reply ios-base/Info.plist:43]: Sure I'll do that in other PR to avoid extending this further

**Review by glm4 — APPROVED** [review]

**Pull request merged**